--- kbd-1.15/src/unicode_start.orig	2008-10-23 21:03:59.000000000 +0200
+++ kbd-1.15/src/unicode_start	2008-12-05 20:17:24.728295331 +0100
@@ -55,7 +55,12 @@
 	# redirect stderr and stdout of loadkeys to /dev/null to avoid the confusing
 	# "plus before udiaeresis ignored" warnings.
 
-	dumpkeys | loadkeys --unicode > /dev/null 2>&1
+	# disabled because it breaks non-iso-8859-1 keymaps
+	# instead use manually:
+	#   loadkeys --unicode <keymap>
+	# or
+	#   dumpkeys -c <charset> | loadkeys --unicode
+	#dumpkeys | loadkeys --unicode > /dev/null 2>&1
 fi
 
 # 2. The output side: the console screen.
